girder_worker/docker/tasks/__init__.py

```diff
@@ -376,3 +377,17 @@ def docker_run(task, image, pull_image=True, entrypoint=None, container_args=Non
     return _docker_run(
         task, image, pull_image, entrypoint, container_args, volumes,
         remove_container, **kwargs)
+
+
+@app.task(base=DockerTask, bind=True)
+def docker_run_chained(task, results, container_args=None, *args, **kwargs):
+    container_args = container_args or []
+    kwargs['container_args'] = [_maybe_transform_chain_result(a, results) for a in container_args]
```

Comment:
I'm a little confused.  You shouldn't need to transform these results in the actual task.  The transformation happens when the task class (`girder_worker.task.Task`) calls `__call__` [here](https://github.com/girder/girder_worker/blob/master/girder_worker/task.py#L149-L170)

Ideally we'd have a ResultTransform associated with the first docker task that returns something like a `GirderFileId` transform.  That should get transformed as apart of the above code block before being passed to the second docker_run task. Is there something i'm missing that is preventing that approach?